contracts/pox-delegation.clar

```diff
@@ -183,38 +174,34 @@
 ;; Users call this function to delegate the stacking rights to a pool.
 ;;
 ;; user-pox-addr: raw bytes of user's address that should be used for payout of rewards by pool admins.
-;; lock-period: desired lock period that pool admin should respect.
 (define-public (delegate-stx (amount-ustx uint) (delegate-to principal) (until-burn-ht (optional uint))
```

Comment:
If lock-period is going to be hardcoded to 1 and users will have unlimited delegation, why is `until-burn-ht` function argument needed at all?

Reply:
Will all users have unlimited delegation? @2snEM6 
Maybe they want to limit their stacking.